examples/classification/configs/sparsity_quantization/mobilenet_v2_imagenet_rb_sparsity_int8.json

```diff
@@ -27,8 +27,8 @@
     "compression": [
         {
             "algorithm": "rb_sparsity", // Compression algorithm name
+            "sparsity_init": 0.01, // Inital sparsity ratio, e.g. value "0.1" means that the method sets 10% of zero weights as a target after the training process starts
             "params": {
-                "sparsity_init": 0.01, // Inital sparsity ratio, e.g. value "0.1" means that the method sets 10% of zero weights as a target after the training process starts
                 "sparsity_target": 0.52, // Desired sparsity ratio, e.g. value "0.5" means that the method will schedule the training so that to get 50% of zero weights in the end
                 "sparsity_target_epoch": 5, // Number of epochs at which sparsity ratio will be increased from "sparsity_init" value up to "sparsity_target" value
                 "sparsity_freeze_epoch": 10 // Overall number of epochs that are used to train sparsity mask
```

Comment:
As @vshampor stated we've  decided to interpret `sparsity_target_epoch` and `sparsity_freeze_epoch` as epoch index.

Reply:
Should probably remove these in-place comments altogether - we have readmes with the same kind of documentation already.

Reply:
Done.